### DISCUSSION TITLE
Nominating @florian-h05 as a Main UI maintainer

### DISCUSSION THREAD

**Discussion by ghys** [openhab/openhab-webui#1812] [WebUI Maintainer Discussions]
Hello @openhab/webui-maintainers!

I hereby endorse @florian-h05 to be a maintainer for the main UI.
Florian has made a number of [quality contributions](https://github.com/openhab/openhab-webui/pulls?q=is%3Apr+author%3Aflorian-h05), understands the overall architecture and writes good code. He's also active in current discussions.

He has accepted the job already and has my full support, I'll still be around but I think it makes perfect sense and will be self-evidently beneficial to have more than one active maintainer, and an additional pair of eyes.

Please cast your votes as always with 👍 or 👎. Thank you!

**Comment by kaikreuzer** [comment]
Hooray, that sounds awesome! 🎉

**Comment by florian-h05** [comment]
Two votes are still missing, but I have already made sure that @ghys has something to do even though I can (hopefully) start helping with reviews soon 😉
I have opened various (small) PRs today that close 4 issues (I am currently trying to fix reported MainUI bugs), but I am also working on a nice enhancement for the search functionality of the developer sidebar.

**Comment by ghys** [comment]
Alright, we now have a majority so I'll leave it to @kaikreuzer to add @florian-h05 to @openhab/webui-maintainers since I don't seem to have the permission.

Once it's done feel free to have a look at, and review, pending PRs which are not yours (feel free to consult me if need be) and I'll have a look at yours.

You're already a maintainer over at openhab-js so you probably know the drill about checking DCOs and that sort of thing. I had an habit of rewriting commit messages, for instance I deemed "[Main UI]" prefixes unnecessary, and I made sure there was a good enough description in that message (not the concatenation of commit messages GitHub offers) and one set of sign-off lines (and a Also-By: line if you assisted and pushed some commits yourself). Also remember to add either "bug" or "enhancement" tags _and_ the "main ui" tag to PRs as well as the current milestone (4.0) so they would be picked up by a script to appear in release notes.

**Comment by kaikreuzer** [comment]
I've added @florian-h05 to the team, welcome!

> I don't seem to have the permission.

Now you should have. 😄

**Comment by florian-h05** [comment]
Thank you all 👍🙏

Yes, I already know the DCO and the other stuff, but thanks for the information anyway!

> I had an habit of rewriting commit messages, for instance I deemed "[Main UI]" prefixes unnecessary, and I made sure there was a good enough description in that message (not the concatenation of commit messages GitHub offers).

FYI: You can configure GitHub to default this commit message to PR title and PR description (you can still modify if they concise enough):

![IMG_1993](https://user-images.githubusercontent.com/73423173/218314398-02dc5085-28b9-4b41-a20c-79de46ac713d.jpeg)

(I can change also this setting if you agree.)

> they would be picked up by a script to appear in release notes.

Out of interest: Where is this script? I could probably use some script at openhab-js as well.

**Comment by kaikreuzer** [comment]
> Where is this script? 

I'm using this here:
https://github.com/kaikreuzer/openhab-scripts

**Comment by ghys** [comment]
> FYI: You can configure GitHub to default this commit message to PR title and PR description (you can still modify if they concise enough)

Didn't know that, thanks, I think it's the best way forward! (in fact I wish I'd known about this earlier 😁)
